addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -527,6 +527,29 @@ const actions = {
         env.call('bus_service', 'startPolling');
         dispatch('_startLoopFetchPartnerImStatus');
     },
+    /**
+     *
+     * Inserts an emoji at the position saved in the store.
+     *
+     * @param {Object} param0
+     * @param {Object} param0.state
+     * @param {string} emoji
+     * @param {string} composerLocalId
+     */
+    insertEmoji(
```

Comment:
rename to `insertEmojiInComposerTextInput ` (clarify this messaging action is on composer text input)

Reply:
Done